internal/pkg/clients/types.go

```diff
@@ -15,12 +15,33 @@ limitations under the License.
 */
 package clients
 
-import "github.com/NethermindEth/sedge/configs"
+import (
+	"github.com/NethermindEth/sedge/configs"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+type ClientType string
+
+func (ct ClientType) ToString() string {
```

Comment:
why not `String` like the standard `ToString`? for Go

Reply:
Fixed in 1741c7fb72189f23933f99ef0e8c1657ee643598